android/java/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxgl/views/MapView.java

```diff
@@ -160,6 +160,30 @@
     // Used to manage Event Listeners
     private ArrayList<OnMapChangedListener> mOnMapChangedListener;
 
+    private enum MapChange {
+        MapChangeNullChange(-1),
+        MapChangeRegionWillChange(0),
+        MapChangeRegionWillChangeAnimated(1),
+        MapChangeRegionIsChanging(2),
+        MapChangeRegionDidChange(3),
+        MapChangeRegionDidChangeAnimated(4),
+        MapChangeWillStartLoadingMap(5),
+        MapChangeDidFinishLoadingMap(6),
+        MapChangeDidFailLoadingMap(7),
+        MapChangeWillStartRenderingFrame(8),
+        MapChangeDidFinishRenderingFrame(9),
+        MapChangeDidFinishRenderingFrameFullyRendered(10),
+        MapChangeWillStartRenderingMap(11),
+        MapChangeDidFinishRenderingMap(12),
+        MapChangeDidFinishRenderingMapFullyRendered(13);
+
+        private int value;
+
+        private MapChange(int value) {
+            this.value = value;
+        }
+    }
+
```

Comment:
>  if we don't need to maintain value for any particular reason it can definitely be safely removed.

I want to maintain equivalence to [this `enum`](https://github.com/mapbox/mapbox-gl-native/blob/1aee3c6c521b502b3c37b87ed28a3140f68e87b2/include/mbgl/map/view.hpp#L14-L29) on the native side.